metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -108,6 +110,24 @@ public void delete(@Nonnull final ConnectorContext requestContext, @Nonnull fina
      * {@inheritDoc}.
      */
     @Override
+    public void update(@Nonnull final ConnectorContext context, @Nonnull final DatabaseInfo databaseInfo) {
```

Comment:
I'd recommend adding Lombok `@NonNull` to these fields so your code is precondition checked to avoid NPE